frontend/src/lib/components/CommandPalette/commandPaletteLogic.ts

```diff
@@ -587,6 +588,19 @@ export const commandPaletteLogic = kea<
                             userLogic.actions.logout()
                         },
                     },
+                    ...(userLogic.values.user?.is_staff ||
+                    userLogic.values.user?.is_debug ||
+                    userLogic.values.user?.is_impersonated
+                        ? [
+                              {
+                                  icon: PlusOutlined,
+                                  display: 'Debug ClickHouse queries',
+                                  executor: () => {
+                                      debugCHQueries()
+                                  },
+                              },
+                          ]
+                        : []),
```

Comment:
This is a bit ugly, would be nicer as a separate command.

Reply:
Hm, userLogic.values won't change though once it's set. I did struggle with this a little so not sure how to make this better